### ISSUE TITLE
Site Forcing SSL on /user/ and /civicrm pages without SSL enabled or any modules enabling it

### ISSUE THREAD

**Issue by zkrebs** [omega8cc/boa#579]
Required log files / config files as per submission guidelines: http://pastebin.com/QPnGYeWP

My domain www.transformationarts.org was in a non-standard directory /distro/oa.004/etc. - I have since moved it to /static/platform

The site is operational, however when I go to www.transformationarts.org/user and log in, it redirects me to a SSL enabled page. I do not have SSL enabled for this site, and I have never had it enabled. Only after the last BOA upgrade, I believe

"Mon Dec 29 20:23:37 EST 2014 / Debian.squeeze x86_64 XEN / Aegir BOA-2.3.8 / Barracuda BOA-2.3.8 / Nginx 1.7.7 / PHP 5.3 / FPM 5.3 / CLI 5.3 / MariaDB-5.5.41 FQDN"

Did this become an issue. 

Why does this matter?

1. Creates scary self-signing certificate warning for my users
2. My civicrm mailings won't have correct read rates because the URL's/image's don't load/ are untrusted

I spoke with the team on another issue and they said it could be my configuration. Possible, but then why did the issue happen only after an upgrade of BOA? 

I thought my CiviCRM version 3.x could be the issue, so I am now on 4.1.x 

EDIT:
Thought it could be an old version of the boost module. Disabled it, no effect.

**Comment by omega8cc** [comment]
There is nothing in BOA which could cause such redirect. Plus, there is no redirect when trying to access it directly, so it must be a site internal issue:

```
quad:~# curl -k -I http://www.transformationarts.org/user
HTTP/1.1 200 OK
Server: nginx
Date: Fri, 23 Jan 2015 13:37:09 GMT
Content-Type: text/html; charset=utf-8
Connection: close
Vary: Accept-Encoding
X-Cookie-Domain: .transformationarts.org
Set-Cookie: NoCacheID=NCBD7e6fe899b0b978ce980fdbe8557f1d77; expires=Fri, 23-Jan-2015 13:37:24 GMT; path=/; domain=.transformationarts.org
X-Limit-Booster: 1
X-Accel-Expires: 1
Last-Modified: Fri, 23 Jan 2015 13:37:09 +0000
Cache-Control: no-cache, must-revalidate, post-check=0, pre-check=0
ETag: "1422020229"
Set-Cookie: SESSf8b574da3e96d712b1d7dc7bd5d781ce=adf16aaaff770b8c2193ac7ee023f3ae; expires=Sat, 24-Jan-2015 13:37:09 GMT; path=/; domain=.transformationarts.org; HttpOnly
X-Device: normal
X-GeoIP-Country-Code: US
X-GeoIP-Country-Name: United States
X-Speed-Cache: BYPASS
X-Speed-Cache-Key: /user
X-NoCache: Skip
X-Server-Name: www.transformationarts.org
Cache-Control: no-store, no-cache, must-revalidate, post-check=0, pre-check=0

quad:~#
```

**Comment by zkrebs** [comment]
I solved the problem.

CiviCRM 3.x was installed, had problem.
Updated to 4.1.x, had problem. Then....

I upgraded to CiviCRM 4.3.9 and it no longer redirects to SSL upon logging in. I have some CiviCRM profiles enabled on /user screens so perhaps that is why it was redirecting. Not BOA's fault per se, but I did have CiviCRM 3.x series running on and it was working fine, so maybe there was a compatibility issue somehow?
